run.sh

```diff
@@ -23,7 +23,6 @@ cd /home/user
 
 # Copy PKGBUILD and *.install scripts
 cp -r "$PKGBUILD_DIR"/* ./ || true
-sed "s|%COMMIT%|$GITHUB_SHA|" "$INPUT_PKGBUILD" > PKGBUILD
```

Comment:
I suspect anyone is depending on this. Perhaps, are you using this elsewhere @FFY00

Reply:
Ah right, didn't know that was a feature. I'm going to close this then.